build/gulpfile.hygiene.js

```diff
@@ -351,6 +338,17 @@ function hygiene(some) {
 		input = some;
 	}
 
+	const tslintSqlConfiguration = tslint.Configuration.findConfiguration('tslint-sql.json', '.');
```

Comment:
Youy don't need to add this to the hygiene run, this will automatically run in the tslint check if you have the rule in the tslint directory and have the rule in the tslint file.

Reply:
If your goal is to limit the scope of the files included for this rule, I would just modify the tslint gulp task to run a second pass with just our lint file, rather than putting this into the hygiene task.

Reply:
You could also see if vscode has interest in taking this rule, since they have the guidance and when they change code they general update any code that doesn't follow the guidance, they probably just don't see this as big enough of a problem to dedicate time to fix.

Reply:
So I didn't add it to the tslint.json because that runs on all files and I didn't want to fix up the vscode ones since that'd just result in a bunch of changes that might cause issues merging down the line. Hence the creation of the new tslint file that will run on just the sql code. (I'll update it later to run on our extensions too - I just didn't want to do the changes now since the fix logic isn't working for some reason so I need to investigate that further and doing the replacements manually is a pain)

I'll follow up with seeing if vscode will take this but for now I figure we can get it in our repo and then remove it later if they decide to take it.